crypto/aes/aes_core.c

```diff
@@ -41,7 +41,7 @@
 #include <stdlib.h>
 #include <openssl/crypto.h>
 #include <openssl/aes.h>
-#include "aes_locl.h"
+#include "aes_local.h"
```

Comment:
TODO: commit message [generated by the script](https://github.com/openssl/openssl/pull/9333/commits/bdea9417a8c3a2fe1cd9643d21b73befc7b41d8f#diff-ca3632d9ac11cc3632fe011cd675e59aR217) is outdated. Needs to be changed to 

```
Rename headers "*_l[o]cl.h" => "*_local.h" [automated]
```